.travis.yml

```diff
@@ -10,12 +10,17 @@ before_install:
 bundler_args: --without development
 
 rvm:
-  - 2.0.0
-  - 2.1.10
-  - 2.2.4
-  - 2.3.0
+  - 2.4
+  - 2.5
+  - 2.6
+jdk:
+  - openjdk8
 matrix:
   include:
-    - rvm: jruby-9.0.5.0
+    - rvm: jruby-9.0
+      env: JRUBY_OPTS="--debug" # for simplecov
+    - rvm: jruby-9.1
+      env: JRUBY_OPTS="--debug" # for simplecov
+    - rvm: jruby
```

Comment:
@drcapulet After reading [JRuby Roadmap](https://github.com/jruby/jruby/wiki/Roadmap), I think we should only support `jruby-9.2` here (which is equivalent to `jruby` for now). Thoughts?

Reply:
Sounds reasonable!